src/cpu.c

```diff
@@ -814,8 +814,7 @@ static int cpu_read (void)
 
 	if (pnumcpu != numcpu || perfcpu == NULL)
 	{
-		if (perfcpu != NULL)
-			free(perfcpu);
+		free(perfcpu);
```

Comment:
Yeah, sfree doesn't have to do that, but it's been like that for more than 10 years. I doubt avoiding the function call has a measurable performance impact in practice.